# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1392622111 18000
#      Mon Feb 17 02:28:31 2014 -0500
# Branch stable
# Node ID 3c1a28978599d65f6b06112d8bcbfce34e95d86a
# Parent  aa7ca90ce746c2eb1aa4fe50b39614899542c1bc
install config.log in $octetc directory

* Makefile.am (octetc_DATA): Include config.log in the list.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -154,18 +154,19 @@ endif
 .PHONY: AUTHORS BUGS INSTALL.OCTAVE
 
 ChangeLog:
 	(cd $(srcdir); hg log --style=build-aux/changelog.tmpl --prune=b0e60ad4ae26 --only-branch=`hg branch`; echo ""; echo "See the files in the directory etc/OLD-ChangeLogs for changes before 2011-04-19") > $@.t
 	mv $@.t $@
 .PHONY: ChangeLog
 
 octetc_DATA = \
+  CITATION \
   NEWS \
-  CITATION
+  config.log
 
 doxyhtml:
 	$(MAKE) -C doc/doxyhtml doxyhtml
 .PHONY: doxyhtml
 
 DIRS_TO_MAKE = \
   $(localfcnfiledir) \
   $(localapifcnfiledir) \
